scripts/setup_ubuntu.sh

```diff
@@ -98,5 +93,5 @@ apt install -yq --allow-downgrades \
 
 # Toolchain variants
 if [[ -n "${SETUP_TOOLCHAIN_VARIANTS}" ]]; then
-  apt install -yq gcc-9 g++-9 clang-10 clang++-10
+  apt install -yq gcc-9 g++-9 clang-10
```

Comment:
Why remove `clang++-10` and not `clang-10`?

Reply:
I was unable to install `clang++-10` locally, it seems like the binary `clang++-10` was provided by `clang-10`.

Reply:
Ah. OK. Reasonable. Just make sure to test the script on 18.04, 20.04, and (as below) maybe 16.04. You can spin up fresh installs on AWS for testing if you need.

Reply:
It is probably easier to use docker to test different distros:

E.g.,
```
docker run --rm -it ubuntu:20.04
```

Anyway, these scripts aren't used for 16.04. That goes through the packaging code:

https://github.com/KatanaGraph/katana-enterprise/tree/master/deploy/package

`scripts/setup_ubuntu.sh` is just used by CI and `setup_dev_ubuntu.sh`